.eslintrc.js

```diff
@@ -33,7 +35,15 @@ module.exports = {
     ],
     'arrow-body-style': ['error', 'always'],
     'import/extensions': ['error', 'always'],
-    strict: ['error', 'global']
+    strict: ['error', 'global'],
+    'jsdoc/require-description': 'warn',
+    'jsdoc/require-param': 'warn',
+    'jsdoc/require-returns': 'warn',
+    'jsdoc/check-tag-names': 'warn',
+    'jsdoc/check-alignment': 'warn',
+    'jsdoc/newline-after-description': 'off', // does not work with 'use strict'
+    'jsdoc/check-indentation': 'warn',
+    'jsdoc/lines-before-block': 'warn'
```

Comment:
I don't think is necessary. 

Where we have something like this: 

```
  return {
      /**
       * currentVersion modifier fetches only the current licence version record for this licence
       */
```

So might be work leaving out.

Reply:
Discussed - this can be an inline comment.